Setup DLAS.bat

```diff
@@ -0,0 +1,78 @@
+@echo off
+:: This file is part of sygil-webui (https://github.com/Sygil-Dev/sygil-webui/).
```

Comment:
Note that I would need to copy their license over as well for this.

Reply:
I can rewrite it, it's just a simple batch file

Reply:
Copying the license is fine, it's compatible, this is more of a note for myself to copy it as required